recipes/prima/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set version = "0.7.1" %}
+
+package:
+  name: prima
+  version: {{ version }}
+
+source:
+  fn: prima-{{ version }}.tar.gz
```

Comment:
You probably don't need this but it doesn't hurt. No strong opinion on removing or keeping it. I'll leave to you.

Reply:
yes, lets remove it if useless